packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -68,7 +68,9 @@
     "holdScreenLabel": "You're on hold",
     "openDtmfDialpadLabel": "Show dialpad",
     "dtmfDialpadPlaceholderText": "Enter number",
-    "outboundCallingNoticeString": "Calling..."
+    "outboundCallingNoticeString": "Calling...",
+    "participantJoinedNoticeString": "Participant joined",
```

Comment:
This is definitely something we should ask about, saying "participant joined" feels like there's so little information its not worth it?
In the bug they mention "Dev has joined the call" which indicates something less generic:
![image](https://user-images.githubusercontent.com/2684369/200673441-e07ccd93-a96b-45d1-a441-72c719faba95.png)

Reply:
Let's talk with Aman about this tomorrow to see if we can keep it generic or if it will need to contain specifics.

Reply:
After talking with Aman, the voice over should indicate who has joined the call rather than voicing out generics.
For example:
"Edward has joined the Call"

Reply:
Did he happen to mention about if multiple people joined the call?